test/benchmarks/include/vikunja/bench/memory.hpp

```diff
@@ -0,0 +1,213 @@
+/* Copyright 2022 Simeon Ehrig
+ *
+ * This file is part of vikunja.
+ *
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#include <alpaka/alpaka.hpp>
+
+#include <algorithm>
+#include <vector>
+
+namespace vikunja::bench
+{
+    template<typename TData>
+    class IotaFunctor
+    {
+    private:
+        TData const m_begin;
+        TData const m_increment;
+
+    public:
+        //! Functor for iota implementation with generic data type.
```

Comment:
```suggestion
        //! Iota functor for generic data types.
```

Also, shouldn't this be part of the class documentation instead of the constructor documentation?